corehq/pillows/xform.py

```diff
@@ -157,6 +162,63 @@ def get_xform_to_elasticsearch_pillow(pillow_id='XFormToElasticsearchPillow', nu
     )
 
 
+def get_xform_pillow(pillow_id='kafka-xform-ucr-es', ucr_division=None,
+                         include_ucrs=None, exclude_ucrs=None,
+                         num_processes=1, process_num=0, ucr_configs=None, skip_ucr=False,
+                         processor_chunk_size=UCR_PROCESSING_CHUNK_SIZE, topics=None, **kwargs):
+    if topics:
+        assert set(topics).issubset(FORM_TOPICS), "This is a pillow to process cases only"
+    topics = topics or FORM_TOPICS
+    change_feed = KafkaChangeFeed(
+        topics, client_id=pillow_id, num_processes=num_processes, process_num=process_num
+    )
+
+    ucr_processor = ConfigurableReportPillowProcessor(
+        data_source_providers=[DynamicDataSourceProvider(), StaticDataSourceProvider()],
+        ucr_division=ucr_division,
+        include_ucrs=include_ucrs,
+        exclude_ucrs=exclude_ucrs,
+    )
+    xform_to_es_processor = ElasticProcessor(
+        elasticsearch=get_es_new(),
+        index_info=XFORM_INDEX_INFO,
+        doc_prep_fn=transform_xform_for_elasticsearch,
+        doc_filter_fn=xform_pillow_filter,
+    )
+    # avoid circular dependency
+    from corehq.pillows.reportxform import transform_xform_for_report_forms_index, report_xform_filter
+    from corehq.pillows.mappings.user_mapping import USER_INDEX
+    xform_to_report_es_processor = ElasticProcessor(
```

Comment:
let's we exclude the 'report form' and 'report case' processors on enterprise clusters

Reply:
You mean, add an option so that these processors are excluded?

Reply:
I mean exclude them if `settings.ENTERPRISE_MODE == True`